### PR TITLE
doc: update for Node.js 15

### PR DESCRIPTION
**Pull request by richardlau**
Add dates to schedule for Node.js 15. Mark Node.js 14 as current.

Schedule SVG regenerated with:

```bash
$ npx lts -s 2020-01-01 -e 2021-12-31 -d "$PWD/schedule.json" -g "$PWD/schedule.svg"
$ npx svgo "$PWD/schedule.svg"
```

### CODE REVIEW

**Comment by targos** [comment]
Can you post a screenshot of the svg?

**Comment by richardlau** [comment]
https://raw.githubusercontent.com/nodejs/Release/049ef91fd3f3cb8b72508f8b58fba22f391afede/schedule.svg
Screen Shot 2020-04-03 at 12 18 54 AM

File: schedule.json [hunk 1]
```diff
@@ -73,5 +73,10 @@
     "maintenance": "2021-10-19",
     "end": "2023-04-30",
     "codename": ""
+  },
+  "v15": {
+    "start": "2020-10-20",
```
Comment (targos) [inline schedule.json:78]: Should we put the 21st instead? To avoid two releases on the same day and make sure 14 will be LTS before the release of 15.

Comment (richardlau) [reply schedule.json]: Updated to the 21st.

Screen Shot 2020-04-22 at 11 57 54 AM

**Pull request merged**